integration-tests/src/test/java/oracle/weblogic/kubernetes/ItResilience.java

```diff
@@ -0,0 +1,179 @@
+import org.awaitility.core.ConditionFactory;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static oracle.weblogic.kubernetes.TestConstants.ADMIN_SERVER_NAME_BASE;
+import static oracle.weblogic.kubernetes.TestConstants.MANAGED_SERVER_NAME_BASE;
+import static oracle.weblogic.kubernetes.TestConstants.MII_BASIC_IMAGE_NAME;
+import static oracle.weblogic.kubernetes.TestConstants.MII_BASIC_IMAGE_TAG;
+import static oracle.weblogic.kubernetes.actions.TestActions.scaleCluster;
+import static oracle.weblogic.kubernetes.utils.CommonMiiTestUtils.createMiiDomainAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkPodReadyAndServiceExists;
+import static oracle.weblogic.kubernetes.utils.OperatorUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+// Test resilience using slammer
+@DisplayName("Test resilience using slammer")
+@IntegrationTest
+class ItResilience {
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static String domainUid = "domain1";
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  private static String adminServerPodName = String.format("%s-%s", domainUid, ADMIN_SERVER_NAME_BASE);
+  private static String managedServerPrefix = String.format("%s-%s", domainUid, MANAGED_SERVER_NAME_BASE);
+  private static int replicaCount = 2;
+  private static LoggingFacade logger = null;
+  private static String ingressHost = null; //only used for OKD
+
+  /**
+   * Perform initialization for all the tests in this class.
+   * Set up the necessary namespaces, install the operator in the first namespace, and
+   * create a domain in the second namespace using the pre-created basic MII image.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *           JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(6, MINUTES).await();
+
+    //install slammer
+    Slammer.installSlammer();
+
+    //check if slammer is up
+    assertTrue(Slammer.list("network"), "Can't reach slammer");
+    // get namespaces
+    assertNotNull(namespaces.get(0), String.format("Namespace namespaces.get(0) is null"));
```

Comment:
You don't need to use String.format() or use String.format("Namespace %s is null", namespaces.get(0))